gnosis/cowswap
# Add hook to generate the affiliate link

# Summary

Closes #849 

Add hook to generate the affiliate link according to https://docs.google.com/document/d/1PsdWn9WPFf2BNeoJNtClyra8jNvWk_bJ-g33TbpUf8s/edit#

The connected Rewards page still needs to be created, I'm not sure if it makes sense to add it to the #951 PR or create a new ticket for it. @alfetopito thoughts?

# Testing steps

~1. Open the app
[ ] In the top menu, there should be a link to the `Rewards` page
2. Click on `Rewards`
[ ] It should lead you to the rewards page
3. Click on the `Connect wallet` button if no wallet is connected. Otherwise click on the `Create affiliate link` button
[ ] The referral link should've been printed on the browser's console~

We are splitting the implementation from the integration so there's no way to test the hook at this point.
